### ISSUE TITLE
Invalid api_user_key error when starting IntelliJ

### ISSUE THREAD

**Issue by jcassee** [kennedyoliveira/ultimate-pastebin-intellij-plugin#16]
Every time I start IntelliJ, I see this error in the logs:
```
Ultimate PasteBin
Error while fetching data from PasteBin
Error: Bad API request, invalid api_user_key
```

When I go to the Ultimate PasteBin settings page and click Check Account Credentials, they turn out to be valid. The Ultimate PasteBin tool window is empty, but when I click the refresh button my pastes are loaded.

Maybe the plugin tries to connect to PasteBin before the credentials are available? Anything else I can do to help debug the problem?

I am using IntelliJ Ultimate Edition 2017.1.1 on Ubuntu 16.04.

**Comment by Mr-Paradox** [comment]
The first thing that you should consider a lot more to do it for a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a few hours ago and it a fK K ew hours ago and it

**Comment by Thibstars** [comment]
I have the same issue, however, when I press the refresh button, my pastes aren't fetched and the same error is shown again.

Clicking on trending pastes and then refreshing seems to do the trick.  Trending pastes as well as personal pastes are then properly shown.

**Comment by 0xbaadf00d** [comment]
Same issue here, but if a click refresh 2 or 3 times, the API token became valid ! Maybe issue is related to timeout settings or Pastebin API itself.

![image](https://cloud.githubusercontent.com/assets/1005086/26814485/3c944350-4a86-11e7-87dd-c55fe4b8fb2f.png)

**Comment by jcassee** [comment]
I have not had this issue since upgrading to IntelliJ 2017.2. @kennedyoliveira Close it?

**Comment by kennedyoliveira** [comment]
Hey @jcassee , thanks for the report and apologies for the long time delay for answering. 
I noticed some problems with 2016 version of intellij, but had no time to investigate, i'll let it open while i do some searching on new version to make sure it's working fine.

**Comment by kennedyoliveira** [comment]
@Thibstars, the trending pastes are a different endpoint in the API and don't need auth, maybe thats why you can fetch the trending but not your own, based on this i imagine is something with the pastebin, since the tests are failing randomly due to the service getting unavailable. Anyway, i'll check to see if there's something wrong.

**Comment by kennedyoliveira** [comment]
Should be fixed by the last version 1.5.2.
